### PR TITLE
Documentation improvement

### PR DESCRIPTION
**Pull request by Zapiano**
A few improvements to the documentation to make it more user friendly - hopefully.

### CODE REVIEW

File: docs/src/development/development_setup.md [hunk 1]
```diff
@@ -15,42 +19,65 @@ In such cases, use the alternate MSIX App installer (https://install.julialang.o
...
-# Add additional debugging tools to sandbox environment
+```julia-REPL
+(sandbox) pkg> add GLMakie GeoMakie GraphMakie
 (sandbox) pkg> add Revise Infiltrator BenchmarkTools JET
-
-# Press ctrl+c to exit the package manager
 ```
 
-Development scripts/functions can then be worked on in the `sandbox` folder, and its sub-folders, without these polluting the ADRIA project itself.
+Press backspace or ctrl+C to leave the package manager.
```
Comment (ConnectedSystems) [inline docs/src/development/development_setup.md:77]:
```suggestion
Press backspace or Ctrl+C to leave the package manager.
```

Or `Control+C`

File: docs/src/index.md [hunk 2]
```diff
@@ -1,61 +1,25 @@
...
-
-# then extract metrics for analysis
-tac = ADRIA.metrics.total_absolute_cover(rs)
-```
+- Running a number of scenarios under different environmental and intervention conditions (see [Loading a Domain](@ref), [Generating scenarios](@ref) and [Running scenarios](@ref));
+- Extracting metrics for each scenario and timestep such as coral cover, shelter volume or number of juveniles (see [Metrics](@ref));
+- Using these metrics to perform some kind of analysis like time series clustering (group together scenarios with similar behaviour) (see [Analysis](@ref));
+- Using the visualization tools to plot some of the mentioned metrics or analysis (see [Analysis](@ref));
 
+More detailes about each of these can be found in the next sections.
```
Comment (ConnectedSystems) [inline docs/src/index.md:14]:
```suggestion
More details about each of these can be found in the next sections.
```

File: docs/src/usage/analysis.md [hunk 3]
```diff
@@ -140,9 +162,11 @@ save("tsa.png", tsa_fig)
 ![Plots of Temporal Sensitivities](/ADRIA.jl/dev/assets/imgs/tsa.png?raw=true "Temporal Sensitivity Analysis")
 
 
-## Time Series Clustering
+### Time Series Clustering
 
-Time Series Clustering generate clusters of time series based on Euclidian distance weighted 
+The Time Series Clustering algorithm generate clusters of data series (tipically time
```
Comment (ConnectedSystems) [inline docs/src/usage/analysis.md:167]:
```suggestion
The Time Series Clustering algorithm generate clusters of data series (typically time
```

File: docs/src/usage/analysis.md [hunk 4]
```diff
@@ -140,9 +162,11 @@ save("tsa.png", tsa_fig)
...
+series) grouping together series with similar behaviour.
+This is achieved by computing the Euclidian distance between each pair of series weighted
```
Comment (ConnectedSystems) [inline docs/src/usage/analysis.md:169]:
```suggestion
The Time Series Clustering algorithm clusters together series (typically time series) 
with similar behavior. 
This is achieved by computing the Euclidian distance between each pair of series weighted
```

Or `generates` -> `identifies` ?

File: docs/src/usage/getting_started.md [hunk 5]
```diff
@@ -28,24 +27,8 @@ Similarly, ADRIA can be updated as new releases are made:
...
-```
-
-## Running ADRIA simulations
-
 Before anything can be done, a `config.toml` file is needed.
-Create a `config.toml` file inside your project directory
-with the following content below (adjusted for your needs).
-
-This `config.toml` file is specific to your computer and project. It **should not be** committed to version control.
+Create a `config.toml` file inside your project directory with the following content below (adjusted for your needs):
```
Comment (ConnectedSystems) [inline docs/src/usage/getting_started.md:31]: This bit should be updated to indicate that a `config.toml` file is optional, and that the assumed default values are shown below.

**Comment by ConnectedSystems** [comment]
TODO:

- [ ] Document ReefModDomain to load and run GBR-scale simulations using ReefModEngine data
- [ ] Examples of the above
- [ ] Double check that the Domain and ResultSet specs are mentioned and linked

**Comment by Zapiano** [comment]
@ConnectedSystems 

1. Just to let you know that I updated the TSC map section
2. Do you have any idea why this PR commits are messed up again...? 😩

**Comment by ConnectedSystems** [comment]
Looks like you merged instead of rebase?

image

**Comment by ConnectedSystems** [comment]
I think all my TODOs are done... lets review and see if we forgot anything.

**Comment by ConnectedSystems** [comment]
Any comments on this before we merge @Rosejoycrocker

**Review by ConnectedSystems — APPROVED** [review]

File: docs/src/architecture/architecture.md [hunk 6]
```diff
@@ -1,2 +1,122 @@
-# ADRIA Architecture
+# Architectural overview
 
+ADRIA consists of three overarching components:
+
+1. a set of Multi-Criteria Decision Analysis methods used to emulate decision-makers
```
Comment (Rosejoycrocker) [inline docs/src/architecture/architecture.md:5]: Extremely picky here, but maybe "simulate" instead of "emulate", emulate suggests a perfect model to me.

**Review by Rosejoycrocker — APPROVED** [review]

**Comment by ConnectedSystems** [comment]
It seems Julia has limited capacity for documenting functors (which is how all metrics are defined). 

There are workarounds but involves more work than I can put in at the moment. Happy to leave this as a limitation until (1) Julia supports documenting these, or (2) I (or one of us) has time to rewrite.

**Review by ConnectedSystems — APPROVED** [review]

**Pull request merged**